snap/snapcraft.yaml

```diff
@@ -0,0 +1,28 @@
+name: maas-cli
+base: core18
+version: git
+summary: CLI to communicate with a MAAS region
+description: |
+  CLI to communicate with a MAAS region providing total automation of your
+  physical servers for amazing data center operational efficiency.
+grade: stable
+confinement: strict
+
+slots:
+  maas-cli:
+    interface: content
+    content: maas-cli
+    read:
+      - $SNAP/lib
```

Comment:
can't we just expose the path of the library?

Reply:
We need all the dependencies not just the libmaas pieces, so I thought it best to expose the whole lib section. Only thing under there is lib/python3.6/site-packages anyway.